scm/src/gmtb_scm_type_defs.f90

```diff
@@ -1648,6 +1795,27 @@ subroutine physics_associate(physics, scm_state, col)
     physics%Sfcprop(col)%tsfc => scm_state%T_surf(col,:)
     physics%Sfcprop(col)%tref => scm_state%T_surf(col,:)
     physics%Sfcprop(col)%slmsk => scm_state%sfc_type_real
+    physics%Sfcprop(col)%vtype => scm_state%veg_type_real
+    physics%Sfcprop(col)%vfrac => scm_state%veg_frac_real
+    physics%Sfcprop(col)%slope => scm_state%slopetype
+    physics%Interstitial(col)%sigmaf = min(scm_state%veg_frac_real,0.01)
```

Comment:
Note to self: Check to see if this is working as expected. The physics%Interstitial DDT gets reinitialized every time step, so this value might be getting cleared out before getting used.